apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -760,7 +760,7 @@ describe('[Chat]', function () {
 				const imgUrlMsgPayload = {
 					_id: `id-${Date.now()}1`,
 					rid: 'GENERAL',
-					msg: 'https://i.picsum.photos/id/671/200/200.jpg?hmac=F8KUqkSzkLxagDZW5rOEHLjzFVxRZWnkrFPvq2BlnhE',
+					msg: 'http://localhost:3000/images/logo/logo.png',
```

Comment:
Perhaps this will not work because I don't think they are trying to download am image per se, but a link that has a preview image.  So I think they are checking the header for the preview, and verifying that the preview is properly rendered in the chat window?